AccessibilitySnapshot/Core/Classes/AccessibilitySnapshotView.swift

```diff
@@ -518,12 +553,162 @@ private extension AccessibilitySnapshotView {
             static let markerSize: CGFloat = 14
             static let markerToLabelSpacing: CGFloat = 16
             static let descriptionLabelToHintLabelSpacing: CGFloat = 4
+            static let labelToCustomActionsSpacing: CGFloat = 4
 
             static let descriptionLabelFont = UIFont.systemFont(ofSize: 12)
             static let hintLabelFont = UIFont.italicSystemFont(ofSize: 12)
 
         }
 
+        private enum Strings {
+
+            static func actionsAvailableText(for locale: String?) -> String {
+                return "Actions Available".localized(
+                    key: "custom_actions.description",
+                    comment: "Description for an accessibility element indicating that it has custom actions available",
+                    locale: locale
+                )
+            }
+
+        }
+
+    }
+
+    // MARK: -
+
+    private final class CustomActionsView: UIView {
+
+        // MARK: - Life Cycle
+
+        init(actionsAvailableText: String?, customActions: [String]) {
+            actionLabels = customActions.map {
+                let iconLabel = UILabel()
+                iconLabel.text = "↓"
```

Comment:
Why use a down arrow for this?

Reply:
I'm trying to reflect how you interact with the element. It will immediately say "Actions Available." You can then perform an up or down flick gesture to iterate through the actions. They're ordered here the same as flicking down through the list (which also matches the order of the actions in the `accessibilityCustomActions` array.